lib/lwm2m_client/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_include_directories(include)
+zephyr_library()
+
+file(GLOB FOTA_SRC "${CMAKE_CURRENT_LIST_DIR}/fota/*.c")
+file(GLOB LWM2M_SRC "${CMAKE_CURRENT_LIST_DIR}/lwm2m/*.c")
+file(GLOB UI_SRC "${CMAKE_CURRENT_LIST_DIR}/ui/*.c")
```

Comment:
There is no reason for using a glob pattern here.
Please use:
```
zephyr_library_sources(<files>)
```

That will also allow you to do:
```
zephyr_library_sources_ifdef(...)
```

```suggestion
```

Reply:
Fixed.